internal/cypress/v1/config.go

```diff
@@ -61,6 +62,7 @@ type Suite struct {
 	Mode             string        `yaml:"mode,omitempty" json:"-"`
 	Timeout          time.Duration `yaml:"timeout,omitempty" json:"timeout"`
 	Shard            string        `yaml:"shard,omitempty" json:"-"`
+	ShardGrepEnabled bool          `yaml:"shardGrepEnabled,omitempty" json:"-"`
```

Comment:
would it be better to add this as a flag in https://github.com/saucelabs/saucectl/blob/main/internal/cmd/run/cypress.go ?

Reply:
Do you mean you think it should be exclusively a cli flag? Or just that I should add a cli flag? I did the latter for now.

Reply:
I mean a cli flag should be added. Just saw you've added one 👍